components/lib/cascadeselect/CascadeSelect.js

```diff
@@ -416,6 +416,8 @@ export const CascadeSelect = React.memo(
             {
                 ref: overlayRef,
                 className: cx('panel'),
```

Comment:
I htink you need to delete this line since you replace className right below it?